src/RequestHandler/RequestHandler.vcxproj

```diff
@@ -231,13 +231,31 @@
     <ClCompile Include="outofprocess\websockethandler.cxx" />
     <ClCompile Include="outofprocess\winhttphelper.cxx" />
   </ItemGroup>
+  <Target Name="CreateVersionHeader" BeforeTargets="PrepareForBuild">
+    <ItemGroup>
+      <VersionHeaderContents Include="// Copyright (c) .NET Foundation. All rights reserved." />
+      <VersionHeaderContents Include="// Licensed under the MIT License. See LICENSE.txt in the project root for license information." />
+      <VersionHeaderContents Include="%0a" />
+      <VersionHeaderContents Include="// This file is auto-generated" />
+      <VersionHeaderContents Include="%0a" />
+      <VersionHeaderContents Include="#define FileVersion $(AspNetCoreModuleVersionMajor),$(AspNetCoreModuleVersionMinor),$(AspNetCoreModuleVersionPatch),$(AssemblyRevision)" />
+      <VersionHeaderContents Include="#define FileVersionStr &quot;$(AspNetCoreModuleVersionMajor).$(AspNetCoreModuleVersionMinor).$(AspNetCoreModuleVersionPatch).$(AssemblyRevision)\0&quot;" />
+      <VersionHeaderContents Include="#define ProductVersion $(AspNetCoreModuleVersionMajor),$(AspNetCoreModuleVersionMinor),$(AspNetCoreModuleVersionPatch),$(AssemblyRevision)" />
+      <VersionHeaderContents Include="#define ProductVersionStr &quot;$(AspNetCoreModuleVersionMajor).$(AspNetCoreModuleVersionMinor).$(AspNetCoreModuleVersionPatch).$(AssemblyRevision)\0&quot;" />
+      <VersionHeaderContents Include="#define PlatformToolset &quot;$(PlatformToolset)\0&quot;" />
+    </ItemGroup>
+    <WriteLinesToFile File="version.h" Lines="@(VersionHeaderContents)" OverWrite="true" />
```

Comment:
Consider adding `WriteOnlyWhenDifferent="True"` to this. Should avoid writing it and breaking increments when the file content hasn't changed. In addition, add the file to writes

```xml
<ItemGroup>
  <FileWrites Include="version.h" />
</ItemGroup>
```

Prevents msbuild from cleaning up the file.